Source/FakeItEasy.Tests/Configuration/ArgumentCollectionTests.cs

```diff
@@ -32,7 +33,7 @@ public void Constructor_that_takes_method_should_set_argument_names()
 
             var arguments = new ArgumentCollection(new object[] { "foo", "bar" }, method);
 
-            Assert.That(arguments.ArgumentNames.SequenceEqual(new string[] { "argument", "argument2" }));
+            arguments.ArgumentNames.ShouldBeEquivalentTo(new[] { "argument", "argument2" });

```

Comment:
*gasp!* You're right. How embarrassing. Will amend.
There are two other `ShouldBeEquivalentTo`s in here (that were originally `Is.EquivalenTo`) that I think should also care about order as well. I will change them all.